programs/programs/connect_for_health/calculator.py

```diff
@@ -45,7 +45,7 @@ def calc_eligibility(self):
                         messages.must_not_have_benefit("Medicaid"))
 
         # Someone has no health insurance
-        has_no_hi = self.screen.has_types_of_insurance(['none'])
+        has_no_hi = self.screen.has_insurance_types(('none', 'private'))
```

Comment:
Why was `private` added here?

Reply:
We want to show Connect for Health if a person has private health insurance. https://docs.google.com/spreadsheets/d/1h-I1oz3l6I7GEHMa2D87XemrxmzDQW1J-mhreFvyqNU/edit#gid=1043594958